src/backend/storeManagers/index.ts

```diff
@@ -91,6 +94,64 @@ export function autoUpdate(runner: Runner, gamesToUpdate: string[]) {
   return gamesToUpdate
 }
 
+export async function sendGameUpdatesNotifications() {
+  const gamesToUpdate: {
+    runner: string
+    game: string
+  }[] = []
+
+  for (const runner in libraryManagerMap) {
+    const games = await libraryManagerMap[runner].listUpdateableGames()
+
+    const gameSettings = await Promise.all(
+      games.map(async (game) => gameManagerMap[runner].getSettings(game))
+    )
```

Comment:
this could take a long time for libraries with 100's of games and it looks like you're only using the first game info below and testing if >1 game to update. 

I also think we should maybe just get HyperPlay games info since those are the games we want to advertise the most

Could you do a filter for HyperPlay games first and then just map the first two results?

Reply:
perfect that saves a lot of computation